src/parsers/xccdf.ts

```diff
@@ -154,7 +201,7 @@ export function processXCCDF(xml: string, removeNewlines: false, useRuleId: 'gro
           }
         }
         if (referenceID && referenceID in ovalDefinitions) {
-          control.descs.check = removeXMLSpecialCharacters(ovalDefinitions[referenceID].metadata[0].title)
+          control.descs.check = removeXMLSpecialCharacters(ovalDefinitions[referenceID].metadata[0].title[0])
```

Comment:
What happens if any of these fields are not populated? Do we need something to the effect of a try catch?